examples/notebooks/bots_on_solidity.ipynb

```diff
@@ -384,57 +421,82 @@
     "    governance,\n",
     ")\n",
     "hyperdrive = project.MockHyperdriveTestnet.at(hyperdrive_address)\n",
+    "print(f\"{dir(hyperdrive)=}\")\n",
     "\n",
+    "print(f\"{fmt(initial_supply/1e18)=}\")\n",
     "with ape.accounts.use_sender(sol_agents[\"agent_0\"]):\n",
     "    base_ERC20.approve(hyperdrive, initial_supply)\n",
     "    as_underlying = True\n",
-    "    hyperdrive.initialize(initial_supply, initial_apr, sol_agents[\"agent_0\"], as_underlying)\n"
+    "    result: Receipt = hyperdrive.initialize(initial_supply, initial_apr, sol_agents[\"agent_0\"], as_underlying)\n",
+    "    hyper_config = hyperdrive.getPoolConfiguration()\n",
+    "    print(f\"{hyper_config=}\")\n",
+    "    pool_info = hyperdrive.getPoolInfo()\n",
+    "    print(f\"{pool_info=}\")\n",
+    "    print(f\"{hyper_config['positionDuration_']/86400=}\")\n",
+    "    spot_price, apr = get_spot_price_and_apr(pool_info, hyper_config, hyperdrive_math)\n",
+    "    print(f\"{fmt(spot_price)=}\")\n",
+    "    print(f\"{fmt(apr)=}\")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "7a2f06f9-193e-4daf-b2bf-75c8111e292f",
-   "metadata": {
-    "tags": []
-   },
+   "id": "62db4b71",
+   "metadata": {},
    "source": [
     "### Execute trades"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "3d90c5d2-6f95-4422-a945-7ecdb0c8e843",
-   "metadata": {
-    "scrolled": true,
-    "tags": []
-   },
+   "id": "5b3e23dc",
+   "metadata": {},
    "outputs": [],
    "source": [
     "# get current block\n",
     "genesis_block_number = ape.chain.blocks[-1].number\n",
     "genesis_timestamp = ape.chain.provider.get_block(genesis_block_number).timestamp\n",
     "\n",
-    "# set the current block?\n",
+    "# set the current block\n",
     "pool_state = [hyperdrive.getPoolInfo().__dict__]\n",
     "pool_state[0][\"block_number_\"] = genesis_block_number\n",
+    "pool_state[0][\"spot_price\"] = spot_price\n",
+    "pool_state[0][\"apr\"] = apr\n",
     "logging.info(\"pool_state=%s\\n\", pool_state)\n",
+    "print(f\"{pool_state=}\")\n",
     "\n",
     "sim_to_block_time = {}\n",
     "trade_receipts = []\n",
-    "for trade in sim_trades:\n",
+    "for trade in sim_trades[1:]:\n",
```

Comment:
avoiding passing the first python trade to solidity, since it's our pool initialize transaction, which was already executed in the solidity `initialize` call